### DISCUSSION TITLE
Some way to detect folder in Pds format

### DISCUSSION THREAD

**Discussion by smoketlt** [dlemstra/Magick.NET#1087] [Help]
Some way to detect folder in Pds format in ImageMagick.NET?  At the moment I use dll photoshop from paint.net for these purposes, but it's too cumbersome. Thanx

**Comment by dlemstra** [comment]
What do you mean by folders?

**Reply by smoketlt** [reply]
![image](https://user-images.githubusercontent.com/95740394/145834887-993b453b-8360-4f80-b049-50b11991cc45.png)

**Reply by smoketlt** [reply]
i mean group with layers

**Reply by dlemstra** [reply]
This is now not possible and I would need to research how and if we can add support for this.
